asm/entities_1.s

```diff
@@ -1,6 +1,111 @@
 .include "asm/macros.inc"
 .include "constants/constants.inc"
 
+.section .rodata
+
+    .global gUnknown_080D550C
```

Comment:
Huh? This is in game code?

Reply:
Oh you moved it here. I understand the idea but whilst this is still asm data it shold remain in the data dir

Reply:
Will leave for now but for next time leave in the data dir for consistency